src/file_resolver.rs

```diff
@@ -86,6 +106,49 @@ impl FileResolver {
         self.files.get(file_no).cloned()
     }
 
+    /// Get the ResolvedFile associated with the `file_no`th `File`
+    pub fn get_resolved_file(&self, file_no: usize) -> Option<ResolvedFile> {
+        self.resolved_files.get(file_no).cloned()
+    }
```

Comment:
Why are we cloning here and not returning a reference?

Reply:
no longer applicable